specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -6365,6 +6365,11 @@
             "$ref": "#/definitions/EnrollmentAccount"
           }
         },
+        "isTransitioned": {
```

Comment:
Unfortunately, api changes like this will need to go in a new api-version.  See the breaking change policy, and api guidelines referenced from the PR checklist